manim/mobject/mobject.py

```diff
@@ -2801,6 +2801,7 @@ class Group(Mobject, metaclass=ConvertToOpenGL):
 
     def __init__(self, *mobjects, **kwargs):
         super().__init__(**kwargs)
+        assert None not in mobjects
```

Comment:
Please add an exception here instead of an assertion. a ValueError with an explicit message would be better IMO.